core/mount/mount_windows.go

```diff
@@ -130,6 +141,40 @@ func (m *Mount) GetParentPaths() ([]string, error) {
 	return parentLayerPaths, nil
 }
 
+// gets the paths of the parent cims of this mount
+func (m *Mount) GetParentCimPaths() ([]string, error) {
+	if m.Type != "CimFS" {
+		return nil, fmt.Errorf("invalid windows mount type: '%s'", m.Type)
+	}
+	var parentLayerCimPaths []string
+	for _, option := range m.Options {
+		paths, found := strings.CutPrefix(option, ParentLayerCimPathsFlag)
+		if found {
+			err := json.Unmarshal([]byte(paths), &parentLayerCimPaths)
+			if err != nil {
+				return nil, fmt.Errorf("failed to unmarshal parent layer cim paths from mount: %w", err)
+			}
+		}
+	}
+	return parentLayerCimPaths, nil
+}
+
+// Only applies to a snapshot created for image extraction, for such a snapshot provides the
+// path to a cim in which image layer will be extracted.
+func (m *Mount) GetCimPath() (string, error) {
+	if m.Type != "CimFS" {
+		return "", fmt.Errorf("invalid windows mount type: '%s'", m.Type)
+	}
+	cimPath := ""
+	for _, option := range m.Options {
+		cimPath, _ = strings.CutPrefix(option, LayerCimPathFlag)
```

Comment:
This will leave `cimPath` set based on whatever was the last element in `m.Options`

Reply:
I'd suggest a utility function:

```go
func getOptionByPrefix(opts []string, prefix string) (string, bool) {
    for _, opt := range opts {
        if val, ok := strings.CutPrefix(opt, prefix); ok {
            return val, true
        }
    }
    return "", false
}
```

```go
cimPath, ok := getOptionByPrefix(opts, LayerCimPathFlag)
if !ok {
	return "", fmt.Errorf("cim path not found")
}
```